package.json

```diff
@@ -150,6 +150,7 @@
     "tsx": "4.19.3",
     "typescript": "5.8.2",
     "unplugin-auto-import": "19.1.1",
+    "v-calendar": "3.1.2",
```

Comment:
Doesn't this break the whole purpose of the UI library which is to avoid depending on any external UI libraries?